platform/linux-dpdk/include/odp/api/feature.h

```diff
@@ -0,0 +1 @@
+../../../../linux-generic/include/odp/api/feature.h
```

Comment:
These changes can go into separate PR as there are many other files that needs to changes as well.